app/Http/Controllers/SurveyController.php

```diff
@@ -0,0 +1,129 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Http\Controllers;
+
+use App\Utils\Upload;
+use App\Survey;
+use App\Organization;
+use Illuminate\Validation\Rule;
+use Illuminate\Http\{Request, JsonResponse};
+use Illuminate\Support\Facades\{Response, DB, URL, Validator};
+use Illuminate\Http\Resources\Json\AnonymousResourceCollection;
+use App\Http\Resources\Survey as SurveyResource;
+
+class SurveyController extends Controller
+{
+
+    /**
+     * @var Upload
+     */
+    protected $upload;
+
+    public function __construct(Upload $upload)
+    {
+        $this->upload = $upload;
+    }
+
+    /**
+     * @return AnonymousResourceCollection
+     */
+    public function index(): AnonymousResourceCollection
+    {
+        return SurveyResource::collection(Survey::paginate(10));
+    }
+
+    /**
+     * @param int $id
+     * @return SurveyResource
+     */
+    public function show(int $id): SurveyResource
+    {
+        return new SurveyResource(Survey::find($id));
+    }
+
+    /**
+     * @param int $id
+     * @return JsonResponse
+     */
+    public function destroy(int $id): JsonResponse
```

Comment:
Move this method to the bottom of this class. If you change the postition of this method, it will be hard to find the method according the controllers